cypress/e2e/wallet.test.ts

```diff
@@ -2,7 +2,7 @@ import { TEST_ADDRESS_NEVER_USE_SHORTENED } from '../support/ethereum'
 
 describe('Wallet', () => {
   before(() => {
-    cy.visit('/')
+    cy.visit('/swap')
```

Comment:
Tests become flaky when there's a redirection here, so I update it to visit `/swap` path directly.

Reply:
ah yeah that works